backend/icpy/services/whatsapp/whatsapp_bot_service.py

```
@@ -0,0 +1,922 @@
+                        if ws_root:
+                            media_files_dir = Path(ws_root) / '.icotes' / 'media' / 'files'
+                            media_files_dir.mkdir(parents=True, exist_ok=True)
+
+                            source_name = os.path.basename(media_path)
+                            dest_name = f"wa_{message_id}_{source_name}"
+                            dest_path = media_files_dir / dest_name
+                            shutil.copy2(media_path, dest_path)
+
+                            stored_abs_path = str(dest_path)
+                            stored_rel_path = f"files/{dest_name}"
+                    except Exception as copy_err:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)